helm/ovn-kubernetes/templates/rbac-ovnkube-node.yaml

```diff
@@ -139,14 +139,17 @@ rules:
       resources:
           - endpointslices
       verbs: [ "get", "list", "watch" ]
+    - apiGroups: ["k8s.cni.cncf.io"]
+      resources:
+          - network-attachment-definitions
+      verbs: ["list", "get", "watch"]
     {{- if eq (hasKey .Values.global "enableInterconnect" | ternary .Values.global.enableInterconnect false) true }}
     - apiGroups: ["networking.k8s.io"]
       resources:
           - networkpolicies
       verbs: [ "get", "list", "watch" ]
     - apiGroups: ["k8s.cni.cncf.io"]
       resources:
-          - network-attachment-definitions
```

Comment:
why did you have to do this? don't follow..

Reply:
ah because of https://github.com/ovn-org/ovn-kubernetes/commit/9e2aa33cf0bce0994ae96a1be982528ee16471a2